include/net/lwm2m.h

```diff
@@ -1145,5 +1145,19 @@ void lwm2m_rd_client_update(void);
  */
 char *lwm2m_path_log_strdup(char *buf, struct lwm2m_obj_path *path);
 
+/* 
```

Comment:
Missing the extra star.
You need `/**` to start a Doxygen block.

Reply:
OK I will fix this.

Reply:
How do I even engage in this zephyr conversation